app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,43 @@
+class ReviewsController < ApplicationController
+
+  def new
+    @shelter = Shelter.find(params[:shelter_id])
+  end
+
+  def create
+    shelter = Shelter.find(params[:shelter_id])
+    new_shelter = shelter.reviews.new(review_params)
+    if new_shelter.save
+      redirect_to "/shelters/#{shelter.id}"
+    else
+      redirect_back(fallback_location: root_path)
+      flash[:error] = "Review must include a title, rating, and content to be submitted."
```

Comment:
If we have time it might be nice to have this display what piece of information was omitted.  I also had tried to find a way to not empty all form data when page reloads.  Minor things, but if we have time something I'd like to revisit.